### PR TITLE
Add EBook compilation to website build + container

### PR DESCRIPTION
**Pull request by dontlaugh**
The build script drops the ebook file in our repository root. We can
move it into the rendered_html folder before archiving. This way, when
the archive is unpacked inside the container, it's located at the
expected root url.

The buildscript should be adjusted to put the ebook in rendered_html
at some point. But this hack works for now.

### CODE REVIEW

**Comment by dontlaugh** [comment]
Richard's recommendation on updating the buildkite steps were basically correct. I just move the ebook into the rendered_html directory before making a tarball. The tarball's folder structure is served statically after it's unpacked inside the container. So adding the ebook in there just works, since the file's set up to serve from the root. References #356

File: .buildkite/pipeline.yaml [hunk 1]
```diff
@@ -4,10 +4,11 @@ steps:
   - label: "Build site"
     key: build
     commands:
-      - "zef install . --deps-only"
-      - "./bin_files/build-site --without-completion --no-status"
-      - "./.buildkite/archive.sh"
-      - "buildkite-agent artifact upload ./raku-doc-website.tar.gz"
+      - zef install . --deps-only
+      - ./bin_files/build-site --without-completion --no-status EBook
+      - mv RakuDocumentation.epub rendered_html
```
Comment (dontlaugh) [inline .buildkite/pipeline.yaml:9]: Only two lines have substantive changes. The others are just me removing the quoted strings to make everything unquoted and consistent.

**Review by coke — APPROVED** [review]
Looks good

**Comment by dontlaugh** [comment]
hmm... The build failed. Gotta look into why that's different up in the build environment vs. my local env.

**Comment by dontlaugh** [comment]
okay something is off with the build envs; i'll need to look at this tomorrow night.

**Comment by finanalyst** [comment]
You must not remove completion in the EBook mode. !!!!!!!

Take out --without-completion

The plugin running in the completion stage here turns the unzipped file
into an epub (zips it with the correct directories).

By contrast in the Website mode, a croapp plugin runs in the completion
stage, and that serves the html files if run locally. So
--without-completion is needed for the build process for website.

2. No need to move the ebook with a bash command. Just adjust the
conf/config-plugin file. The name of the ebook and its path are set there.
The file is something like 'EBook/configs/3-plugin-configs' and the field
should be obvious

**Comment by finanalyst** [comment]
@dontlaugh There are two steps needed:
- Build the website, this renders all sources into HTML, constructs secondaries, creates index pages, etc
- Build the EBook. this renders all sources into XHTML, constructs the infrastructure files needed for EBook navigation

Both need to happen. It seems to me, unless I have misunderstood, that you are only running one (EBook) and removed the build step of the Website.

The syntax is `build-site <options> MODE` where MODE is 'EBook' or 'Website'
<options> for Website should include `--without-completion` options for EBook should not.

**Comment by dontlaugh** [comment]
I see now. I only had to invoke the build script once locally, but that was because the static site had already been run.

**Comment by dontlaugh** [comment]
@finanalyst Thank you for updating the scripts and config for me!

I have deployed this change to prod, and the ebook is available for download now.

**Pull request merged**